README.md

```diff
@@ -63,3 +63,18 @@ Head over to [Quickstart Flutter](https://github.com/hypertrack/quickstart-flutt
     1. flutter pub publish --dry-run
         - to test is everything OK with release
     2. flutter pub publish
+
+## How to change build config
+1. compileSdkVersion
+  - change it in android/build.gradle
+  - android {
+    compileSdkVersion 
```

Comment:
That doesn't look right. It should be a markdown and contain something like

```
android {
    ...
    compileSdkVersion ${putYourNumberHere}
    ...
}
```

Reply:
+1

Reply:
TBD (out of scope of this PR)